packages/transaction-controller/src/helpers/MultichainTrackingHelper.ts

```diff
@@ -190,23 +190,11 @@ export class MultichainTrackingHelper {
     if (!this.#isMultichainEnabled) {
```

Comment:
Can this entire method just be `new EthQuery(getProvider(networkClientId, chainId)`?

Since the same logic now exists in `getProvider`.

Reply:
Good call Matthew, I will just return as you mentioned.
It's now a one-liner method. But I will keep it in order to not do many changes in the places it's been used, if that's ok. 

Fixed in https://github.com/MetaMask/core/pull/4055/commits/bfe24c3f198295f4aa80b8f42bbb2e74bdc1664d

Reply:
Done in https://github.com/MetaMask/core/pull/4055/commits/bfe24c3f198295f4aa80b8f42bbb2e74bdc1664d